src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTask.java

```diff
@@ -65,6 +68,10 @@ private void upload(Letter letter) throws IOException {
             zippedDoc.filename
         );
 
-        ftp.upload(zippedDoc);
+        ftp.upload(zippedDoc, isSmokeTest(letter));
+    }
+
+    private boolean isSmokeTest(Letter letter) {
+        return Objects.equals(letter.getType(), SMOKE_TEST_LETTER_TYPE);
```

Comment:
I like `SMOKE_TEST_LETTER_TYPE.equals(letter.getType())` more, but it's really a matter or preference.